source/io/module_javaScriptInvoke.js

```diff
@@ -79,28 +81,26 @@
 
         // Private Instance Variables (per vireo instance)
         var internalFunctionsMap = new Map();
-        var JavaScriptInvoke_GetParameterType = Module.cwrap('JavaScriptInvoke_GetParameterType', 'number', ['number', 'number']);
-        var JavaScriptInvoke_GetParameterPointer = Module.cwrap('JavaScriptInvoke_GetParameterPointer', 'number', ['number', 'number']);
-        var JavaScriptInvoke_GetArrayElementType = Module.cwrap('JavaScriptInvoke_GetArrayElementType', 'number', ['number']);
+        var JavaScriptInvoke_GetParameterTypeRef = Module.cwrap('JavaScriptInvoke_GetParameterTypeRef', 'number', ['number', 'number']);
```

Comment:
Because these are only numerics in and out, we can avoid the overhead of using cwrap. You can call JavaScriptInvoke_GetParameterTypeRef by doing:

```
var resultNum = Module._JavaScriptInvoke_GetParameterTypeRef(num1, num2);
```

Note the c function name is prefixed with an underscore when accessed directly from the Module object.

Reply:
Similar to how you did the TypeRef call: https://github.com/ni/VireoSDK/blob/aa4422ca3aa849cefdb1442fd09f55f016545845/source/core/module_typeHelpers.js#L288

Reply:
ah, I see. good to know. Done.